byterun/win32.c

```diff
@@ -180,12 +180,12 @@ CAMLexport caml_stat_string caml_search_exe_in_path(char * name)
   if (fullnamelen < 256) fullnamelen = 256;
   while (1) {
     fullname = caml_stat_alloc(fullnamelen);
-    retcode = SearchPath(NULL,              /* use system search path */
-                         name,
-                         ".exe",            /* add .exe extension if needed */
-                         fullnamelen,
-                         fullname,
-                         &filepart);
+    retcode = SearchPathA(NULL,              /* use system search path */
+                          name,
+                          ".exe",            /* add .exe extension if needed */
+                          fullnamelen,
+                          fullname,
+                          &filepart);
```

Comment:
Yes, will do it, but since it is used in the implementation of `execv*` and `createprocess.c`, it is best to adapt all that code at once.